registry/registry.dsc.resource.json

```
@@ -1,7 +1,7 @@
 {
   "$schema": "https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/08/bundled/resource/manifest.json",
   "type": "Microsoft.Windows/Registry",
-  "description": "Registry configuration provider for the Windows Registry",
+  "description": "Registry configuration adapter for the Windows Registry",
```

I think this is more accurately something like:

```suggestion
  "description": "Manage Windows Registry keys and values",
```

This resource is not an adapter.